web-frontend/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "codemirror": "^5.65.2"
+  }
```

Comment:
Instead of committing the node_modules, we can have them installed on demand.

Also, if we add a scripts section running `npm start` can install the node dependencies from the package and package-lock files (rather than having them in git) also start that python server.

```json
  "scripts": {
    "start": "npm install && python3 -m http.server 80"
  }
```